response_operations_ui/views/social.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from flask import Blueprint, render_template, request
+from flask_login import login_required
+from structlog import wrap_logger
+
+from response_operations_ui.controllers.case_controller import get_cases_by_sample_unit_id
+from response_operations_ui.controllers.sample_controllers import search_samples_by_postcode
+from response_operations_ui.forms import SearchForm
+
+logger = wrap_logger(logging.getLogger(__name__))
+social_bp = Blueprint('social_bp', __name__,
+                      static_folder='static', template_folder='templates')
+
+
+@login_required
+@social_bp.route('/', methods=['GET', 'POST'])
+def social_case_search():
+    form = SearchForm()
```

Comment:
Maybe we should use another form with postcode validation?

Reply:
Would be nice but I expect we'd have to roll our own 😱